File: esp-src/esp-be/src/main/java/edu/esp/be/Student.java
package edu.esp.be;

public class Student {
    public String ID;
    public String password;
    public String newPassword;
    public String confirmNewPassword;
    public String fullName;
    public String SSN;
    public String dateOfBirth;
    public String address;
    public String phone;
    public String landline;
    public String email;
    public String gender;

    @Override
    public String toString() {
        return "Student{" +
                "ID='" + ID + '\'' +
                ", password='" + password + '\'' +
                ", newPassword='" + newPassword + '\'' +
                ", confirmNewPassword='" + confirmNewPassword + '\'' +
                ", fullName='" + fullName + '\'' +
                ", SSN='" + SSN + '\'' +
                ", dateOfBirth='" + dateOfBirth + '\'' +
                ", address='" + address + '\'' +
                ", phone='" + phone + '\'' +
                ", landline='" + landline + '\'' +
                ", email='" + email + '\'' +
                ", gender='" + gender + '\'' +
                '}';
    }
}

File: esp-src/esp-fe/src/app/System Entities/Student.ts
export class Student{

  studentId: string
  dptId: string
  studentLevel: string
  gpa: string
  studentName: string
  ssn: string
  bdate: string
  studentAddress: string
  phone: string
  landline: string
  gender: string
  email: string

  constructor(studentId: string, dptId: string, studentLevel: string, gpa: string,
              studentName: string, ssn: string, bdate: string, studentAddress: string, phone: string,
              landline: string, gender: string, email: string){

    this.studentId = studentId
    this.dptId = dptId
    this.studentLevel = studentLevel
    this.gpa = gpa
    this.studentName = studentName
    this.ssn = ssn
    this.bdate = bdate
    this.studentAddress = studentAddress
    this.phone = phone
    this.landline = landline
    this.gender = gender
    this.email = email

  }
}
